Minio.Examples/Cases/PutObject.cs

```diff
@@ -32,14 +34,22 @@ class PutObject
             {
                 byte[] bs = File.ReadAllBytes(fileName);
                 System.IO.MemoryStream filestream = new System.IO.MemoryStream(bs);
```

Comment:
When does `filestream`'s resources get finalized? Would it be convenient to use `using` statement to concisely handle resource finalization?